tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/ArchetypesTest.java

```diff
@@ -53,9 +67,27 @@
  * @contributor jkopriva@redhat.com
  * @contributor vprusa@redhat.com
  */
+@JRE(cleanup=false)
 @RunWith(RedDeerSuite.class)
 public class ArchetypesTest {
 
+	@RequirementRestriction
+	public static Collection<RequirementMatcher> getRestrictionMatcher() {
+		return Arrays.asList(new RequirementMatcher(JRE.class, "version", "1.8"));
+	}
+	
+	@Retention(RetentionPolicy.RUNTIME)
+	@Target(ElementType.TYPE)
+	public @interface JRE {
+
+		/**
+		 * Cleanup.
+		 *
+		 * @return true, if successful
+		 */
+		boolean cleanup() default false;
+	}
+	
 	private static final String CENTRAL_LABEL = "Red Hat Central";
 	private static final String MAVEN_SETTINGS_PATH = System.getProperty("maven.config.file") == null
```

Comment:
I think this maven settings file support has been removed some time ago, please remove it too.

Reply:
Removed.